src/main/java/core/basesyntax/Splitter.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Splitter {
+    public ReportItem splitString(String string) {
+        String[] stringArray = string.split(",");
```

Comment:
Consider using constants for delimiter strings like "," to improve code readability and maintainability.